### ISSUE TITLE
[docs] Using consistent models in documentation

### ISSUE THREAD

**Issue by ioneyed** [Permify/permify#361]
**Is your feature request related to a problem? Please describe.**
The documentation for how to model or create relationships appears to be incorrect or disjointed with previous context from the page.

Given this page: https://docs.permify.co/docs/getting-started/sync-data#more-relation-tuple-examples

In this page the model

```
entity user {} 

entity organization {

    relation member @user

} 

entity document {
    
    relation  owner  @user   
    relation  org    @organization      

    action view   = owner or org.member
    action edit   = owner 
    action delete = owner
} 
```

is used above the section of more relation-tuple-examples but the examples do not relate to that model or the provided model for that example isn't given making the reader scratching their head a bit on the connection.

For instance - further down the page there is the **Organization Members are viewer of Repo** which presents this API payload

```
{
    "entity": {
        "type": "repository",
        "id": "1"
    },
    "relation": "viewer",
    "subject": {
        "type": "organization",
        "id": "2",
        "relation": "member"
    }
}
```
With the caption of 

>Created relational tuple: repository:1#admin@organization:2#member
>Definition: Members of organization 2 are viewers of repository 1.

However, without the model for this example I would say that the API call and the caption are incorrect because it says that the members of Organization 2 should be **viewers** of repository 1 but it clearly says **admin** in the caption while the payload says **viewer**.

**Describe the solution you'd like**

I think it would benefit the user if the documentation was based around simple and complex models for people to relate to the documentation. Either completely separate pages with the model presented first and a description of the use case or with each example you provide a tab UI component that provides the model, the data, and the api call (or nix the API call except in the API/SDK documentation) or potentially a link to the playground to test the scenario with the relevant data/model attached.

**Describe alternatives you've considered**

Ignoring the documentation but that isn't as fruitful as trying to seek change :D

**Additional context**

On a product perspective with IAM space - Documentation, examples, analogies, and content will be the engine to drive successful adoption in larger enterprises who may not be on the up and up of IAM controls. Having documentation, testimonies, and "real life" examples of the permission models with examples that go in depth will drive more people to this over OpenFGA from Auth0 or other solutions that pop up.

**Comment by EgeAytin** [comment]
Hi @ioneyed, thank you for your feedback. I appreciate you pointing out the error in the tuple repository:1#admin@organization:2#member, it should be repository:1#viewer@organization:2#member.

Regarding the "More Relational Tuples" section, I want to clarify that it was not intended to be a continuation of the previous model, but rather to provide different examples of relational tuples to showcase their structure and potential use cases. However, upon further review and consideration of your feedback, we have decided to do it as you suggested. So we'll connect each relational tuple with its respective model to provide a better understanding of how they are used and make it easier to test them with adding api calls and link with playground. 

We plan to implement this change later this week.

**Specifications:**
* Updating the "More Relational Tuples" section with providing a tab UI component for each example model.
* Each example Model (tab component) will consist; schema, the data, and the api call (or nix the API call except in the API/SDK documentation) and link with playground.
* Fix the issue in the tuple: `repository:1#admin@organization:2#member` it should be `repository:1#viewer@organization:2#member`

**Comment by EgeAytin** [comment]
Hi @ioneyed, I wanted to let you know that we have updated the page at https://docs.permify.co/docs/getting-started/sync-data. We have made changes to the example schema and have related the relational tuple examples with the schema. Additionally, we have added API calls to each tuple and provided a playground link for the entire example.

Moving forward, we are working on a separate page that will provide real-world and more depth examples. We plan to combine this page with the use cases page to provide a better view of what can be achieved with Permify.
